tests/utils/common/common.py

```diff
@@ -693,6 +693,15 @@ def version_is_minimum(bitbake_variables, component, min_version, recurse=True):
     return version_parsed >= packaging.version.Version(min_version)
 
 
+def version_is_minimum_cpp_client(bitbake_variables, component):
```

Comment:
Shouldn't this be called `is_cpp_client`:

* The function won't work correctly if `component != "mender"`, so it should not take this argument.
* There is no `version` argument, so the `version_is_minimum` is misleading, IMHO.

I agree with its function though, it's a good solution.